README.md

```diff
@@ -97,6 +97,27 @@ bulkinvalidaddresses
 Rows which are successfully processed will be added to `PROCESSED_invalid_addresses_*.csv` and errored rows be appended to `ERROR_invalid_addresses_*.csv` with the corresponding error details written to `ERROR_DETAIL_invalid_addresses_*.csv`.
 
 
+### Find Invalid Address Case IDs from UPRN File
+Run Book - https://collaborate2.ons.gov.uk/confluence/display/SDC/Find+Invalid+Address+Case+ID%27s+by+UPRN+-+ADDRESS_DELTA
+
+When we receive a file of UPRNs for cases that have been identified as invalid addresses, this feature will call the Case API against the UPRNs provided and generate a new file in the bulk invalid address bucket with the name `invalid_addresses_invalid_uprn.csv`.
+
+This is done by running:
+```shell script
+invalidaddressdelta <file_name>
+```
+
+The rows which are successfully processed will be added to `PROCESSED_invalid_addresses_invalid_uprns.csv`
```

Comment:
I would change this to say it creates a file matching `invalid_addresses_*.csv` so it can be picked up by the bulkinvalidaddresses script.